Lesson4/src/by/home/work/Computer.java

```diff
@@ -0,0 +1,74 @@
+package by.home.work;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Computer {
+
+    private String cpu;
+    private int ram;
+    private int hdd;
+    private boolean work;
+
+    public Computer() {
+
+    }
+
+    public Computer(String cpu, int ram, int hdd) {
+        this.cpu = cpu;
+        this.ram = ram;
+        this.hdd = hdd;
+    }
+
+    public void info() {
+        System.out.println("I have computer at home : " + cpu + " " + "RAM : " + ram + " " + " HDD : " + hdd);
+    }
+
+    public void push() {
+        Random random = new Random();
+        Scanner scanner = new Scanner(System.in);
```

Comment:
so it seems like your code allows to enable PC that is already burned. I can call push twice, on the first attempt I won't pass random check, on the second attempt, I will pass it and will turn on PC. It shouldn't happen

Also you might want to implement turn off method